templates/components/products/product-view.html

```diff
@@ -248,9 +248,7 @@ <h2 class="productView-brand"{{#if schema}} itemprop="brand" itemscope itemtype=
                 {{> components/common/wishlist-dropdown}}
             {{/if}}
         </div>
-        {{#unless is_ajax}}
```

Comment:
No longer needed to hide from AJAX (quick view) as we no longer have a JS dependency, and the sharing URL is passed explicitly from the product object.